crates/entities/src/models/crawl_queue.rs

```diff
@@ -367,46 +369,64 @@ pub async fn enqueue_all(
         })
         .collect();
 
+    // Ignore urls already indexed
+    let mut is_indexed: HashSet<String> = HashSet::with_capacity(urls.len());
     // Igore urls already indexed
-    let is_indexed: HashSet<String> = indexed_document::Entity::find()
-        .filter(indexed_document::Column::Url.is_in(urls.clone()))
-        .all(db)
-        .await?
-        .iter()
-        .map(|x| x.url.to_string())
-        .collect();
+    for chunk in urls.chunks(10000) {
```

Comment:
Could we pull out the # of chunks into a constant at the top of the file?

Reply:
I went with 10k.